google/cloud/security/scanner/scanner.py

```diff
@@ -69,13 +69,50 @@
                      'the format of the path should be '
                      '"gs://bucket-name/path/for/output".'))
 
+flags.DEFINE_bool('list_engines', False, 'List all rule engines')
+
+flags.DEFINE_string('use_engine', None, 'Which engine to use')
```

Comment:
Yep, so either it should be scripted to go through all the engines or we can improve it even more to go through the whole map.